README.md

```diff
@@ -1,124 +1,10 @@
-# Automated Prometheus E2E testing and benchmarking.
+## Intro
+The repository provides two main functions:
+- fucnbench
+- prombench
 
-![Prombench Design](design.svg)
+## Prombench
+Benchmarking of the whole Prometheus stack and comparison with the Prometheus master branch. The benchmark is initiated by commenting on a pull request in the Prometheus repository. See Prombench [README.md](/cmd/prombench/README.md) for more information.
```

Comment:
```suggestion
Benchmarking of the whole Prometheus stack and comparison or a PR with another Prometheus release. The benchmark is initiated by commenting on a pull request in the Prometheus repository. See Prombench [README.md](/cmd/prombench/README.md) for more information.
```

Reply:
I don't understand the meaning of the sentence then.

Reply:
I updated it a bit, is this more clear?

Reply:
I updated it again to simplify, no need to include more details as these should be mentioned in the extended readme. It will avoid having to modify if the design changes at some point.